src/rt/mod.rs

```diff
@@ -43,6 +43,10 @@ where
     F: FnOnce() + 'static,
 {
     execution(|execution| {
+        if execution.log {
+            println!("spawn");
```

Comment:
Similarly to my other comments, I think it would be very helpful to include something identifying the thread to these messages. I think we can use the `loom` internal object ID for this.

Reply:
Done.